AdminWebsite/AdminWebsite/ClientApp/src/app/booking/hearing-schedule/hearing-schedule.component.html

```diff
@@ -114,8 +114,8 @@ <h2 class="govuk-heading-m">Time and location</h2>
           <span id="hearingDateWeekend-error" class="govuk-error-message"> Select a date </span>
         </div>
 
-        <div *ngIf="addHearingDateControl.errors?.weekend || addHearingDateControl.errors?.publicHoliday" class="alert alert-danger">
-          <span id="hearingDateWeekend-error" class="govuk-error-message"> Select a working day </span>
+        <div *ngIf="addHearingDateControl.errors?.weekend" class="alert alert-danger">
+          <span id="hearingDateWeekend-error" class="govuk-error-message"> Select a day from Monday to Friday </span>
```

Comment:
What is this for?

Reply:
this one is merged from another story to exclude the hearing dates should not be on weekends/band holidays